lib/services/webSiteManagement2/lib/models/address.js

```diff
@@ -1,9 +1,5 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- * 
- * Code generated by Microsoft (R) AutoRest Code Generator 0.15.0.0
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.16.0.0
```

Comment:
@lmazuel - I thought "header" : "MICROSOFT_MIT_NO_VERSION" will have the header text and will not have the autorest version.
However, I see exactly opposite over here.